src/lib/tls/tls_handshake_io.h

```diff
@@ -204,11 +216,11 @@ class Datagram_Handshake_IO final : public Handshake_IO
       uint64_t m_last_write = 0;
       uint64_t m_next_timeout = 0;
 
-      uint16_t m_in_message_seq = 0;
```

Comment:
Why this change?

Reply:
Ugh, this is to fix the GCC -Wreorder warning in CI.

Reply:
OK